v3/src/components/menu-bar/user-entry-modal.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button } from "@chakra-ui/react"
+import React from "react"
+import { useCfmContext } from "../../hooks/use-cfm-context"
+import { t } from "../../utilities/translation/translate"
+
+import "./user-entry-modal.scss"
+
+interface IProps {
+  isOpen: boolean
+  onClose: () => void
+}
+
+export const UserEntryModal = ({ isOpen, onClose }: IProps) => {
+  const cfm = useCfmContext()
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLDivElement>) => {
+    if (event.key === "Escape") {
+      onClose()
+    }
+    if (event.key === "Enter") {
+      openDocument()
+    }
+  }
+
+  const openDocument = () => {
+    cfm?.client.openFileDialog()
+    onClose()
+  }
+
+  const createNewDocument = () => {
+    onClose()
+  }
+
+  const buttons = [{
+    label: t("DG.main.userEntryView.openDocument"),
+    onClick: openDocument,
+    default: true
```

Comment:
The two buttons do not appear different so the `default` doesn't seem to be working.